src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+import { MovieListProps } from '../../types/MovieListProps';
+import { MovieCard } from '../MovieCard';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+export const MoviesList: React.FC<MovieListProps> = ({ movies }) => {
+  const key = uuidv4();
```

Comment:
we have imdbId... it seems special for key

Reply:
Yeah, you have id for it, but it's good that you try to use something new for you
But, if you have id - you don't need this libs